docs/release-notes/2.6.0.md

```diff
@@ -23,7 +23,7 @@ Percona Monitoring and Management (PMM) is a free and open-source platform for m
 ## New Features
 
 - [PMM-5728](https://jira.percona.com/browse/PMM-5728): Technical preview of External Services monitoring feature. A new command provides integration with hundreds of third-party systems ([https://prometheus.io/docs/instrumenting/exporters/](https://prometheus.io/docs/instrumenting/exporters/)) via the Prometheus protocol so that you can monitor external services on a node where PMM agent is installed.
-- [PMM-5822](https://jira.percona.com/browse/PMM-5822): PMM now includes a Security Threat Tool to help users avoid the most common database security issues. Read more [here](../using/security-threat-tool.md).
```

Comment:
Thought it best to simply remove the link here, not seeing an equivalent any longer. Besides, these RNs are old.